alerts/openshift-container-storage-operator/HighRBDCloneSnapshotCount.md

```diff
@@ -0,0 +1,68 @@
+# HighRBDCloneSnapshotCount
+
+## Meaning
+
+The number of RBD clones or snapshots (children) of a Ceph RBD image has
+exceeded 200. This metric is derived from CSI volume metadata and reflects
```

Comment:
`CSI volume metadata` This is derived from rbd children count right mot volume metadata